apps/webapp/app/components/runs/RunStatuses.tsx

```diff
@@ -169,3 +177,35 @@ export const filterableStatuses: Record<FilterableStatus, JobRunStatus[]> = {
 };
 
 export const statusKeys: FilterableStatus[] = Object.keys(filterableStatuses) as FilterableStatus[];
+
+export const FilterableRelativeTimeFrame = z.union([
+  z.literal("5 mins"),
+  z.literal("15 mins"),
+  z.literal("30 mins"),
+  z.literal("1 hour"),
+  z.literal("3 hours"),
+  z.literal("6 hours"),
+  z.literal("1 day"),
+  z.literal("3 days"),
+  z.literal("7 days"),
+  z.literal("10 days"),
+  z.literal("14 days"),
+  z.literal("30 days"),
+]);
+
+export type FilterableRelativeTimeFrame = z.infer<typeof FilterableRelativeTimeFrame>;
+
+export const relativeTimeFrameKeys: FilterableRelativeTimeFrame[] = [
+  "5 mins",
+  "15 mins",
+  "30 mins",
+  "1 hour",
+  "3 hours",
+  "6 hours",
+  "1 day",
+  "3 days",
+  "7 days",
+  "10 days",
+  "14 days",
+  "30 days",
+];
```

Comment:
Given that these are details that are only related to the relative time filtering component, I think it makes more sense for them to not be in here.

In fact, I think it'd be better to just have a const and infer the type from that (in the other file).

Like this:

```ts
const timeFrameValues = [
  {
    label: "5 mins",
    value: 5 * 60 * 1000,
  },
  {
    label: "15 mins",
    value: 15 * 60 * 1000,
  },
  {
    label: "30 mins",
    value: 30 * 60 * 1000,
  },
  {
    label: "1 hour",
    value: 60 * 60 * 1000,
  },
  {
    label: "3 hours",
    value: 3 * 60 * 60 * 1000,
  },
  {
    label: "6 hours",
    value: 6 * 60 * 60 * 1000,
  },
  {
    label: "1 day",
    value: 24 * 60 * 60 * 1000,
  },
  {
    label: "3 days",
    value: 3 * 24 * 60 * 60 * 1000,
  },
  {
    label: "7 days",
    value: 7 * 24 * 60 * 60 * 1000,
  },
  {
    label: "10 days",
    value: 10 * 24 * 60 * 60 * 1000,
  },
  {
    label: "14 days",
    value: 14 * 24 * 60 * 60 * 1000,
  },
  {
    label: "30 days",
    value: 30 * 24 * 60 * 60 * 1000,
  },
];

type RelativeTimeFrameItem = typeof timeFrameValues[number];
```